prow/oss/terraform/modules/alerts/main.tf

```diff
@@ -141,3 +141,40 @@ resource "google_monitoring_alert_policy" "heartbeat-job-stale" {
   # gcloud beta monitoring channels list --project=oss-prow
   notification_channels = ["projects/${var.project}/notificationChannels/${var.notification_channel_id}"]
 }
+
+resource "google_monitoring_alert_policy" "webhook-missing" {
+  project      = var.project
+  display_name = "webhook-missing"
+  combiner     = "OR" # required
+
+  conditions {
+    display_name = "webhook-missing"
+
+    condition_monitoring_query_language {
+      duration = "720s"
+      query    = <<-EOT
+      fetch k8s_container
+      | metric 'workload.googleapis.com/prow_webhook_counter'
+      | sum
+      | align delta_gauge()
```

Comment:
nit: I don't think this needs to be converted to a gauge, what is the motivation?

Reply:
when I uses `delta` on UI, the json generated at the last line is something like `val() == 0 'no_unit_value'`, changing this to gauge just for avoiding the annoyance